PWGHF/TableProducer/candidateCreatorCascade.cxx

```diff
@@ -330,7 +334,7 @@ struct HfCandidateCreatorCascadeMc {
             auto daughI = particlesMC.rawIteratorAt(arrDaughLcIndex[iProng]);
             arrDaughLcPDG[iProng] = daughI.pdgCode();
           }
-          if (!(arrDaughLcPDG[0] == arrDaughLcPDGRef[0] && arrDaughLcPDG[1] == arrDaughLcPDGRef[1] && arrDaughLcPDG[2] == arrDaughLcPDGRef[2])) { // this should be the condition, first bach, then v0
+          if (!(arrDaughLcPDG[0] == sign * arrDaughLcPDGRef[0] && arrDaughLcPDG[1] == arrDaughLcPDGRef[1] && arrDaughLcPDG[2] == arrDaughLcPDGRef[2])) { // this should be the condition, first bach, then v0
```

Comment:
Do you not need `sign` for all daughters?

Reply:
No, because the $\pi^+$ and the $\pi^-$ do not swap place. They always come from a decay of a $K^0_S\to \pi^++\pi^-$.
The decay chains are
$L_c^+\to p+K^0 \to p + K^0_S \to p + \pi^+ + \pi^-$
$L_c^-\to \bar{p} + \bar{K^0} \to \bar{p} + K^0_S \to \bar{p} +\pi^++\pi^-$

Reply:
OK, if the positive pion always comes first, then it should be fine. Thanks